parlai/agents/image_seq2seq/image_seq2seq.py

```diff
@@ -45,18 +45,9 @@ def add_cmdline_args(cls, argparser):
         """
         Override to add one arg.
         """
-        super(ImageSeq2seqAgent, cls).add_cmdline_args(argparser)
+        TransformerGeneratorAgent.add_cmdline_args(argparser)
+        TorchImageAgent.add_cmdline_args(argparser)
```

Comment:
Can you try just super().add_cmdline_args(argparser)? You might have to do it in both places.

This is an experiment.

Reply:
Hmm when I do this I get `Parse Error: unrecognized arguments: --image-encoder-num-layers 1 --image-features-dim 2048`, indicating that it couldn't find the `TorchImageAgent` args